package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lifeomic/graphql-resolvers-xray-tracing",
-  "version": "2.0.0",
+  "version": "2.2.0",
```

Comment:
I would avoid bumping the version number until the changes are merged

Reply:
unbumped